Chat/package.json

```diff
@@ -18,8 +18,7 @@
   },
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
-    "@azure/communication-calling": "1.4.4",
```

Comment:
Is calling SDK deleted by accident?

Reply:
No. I don't think this is used. This is just a chat sample so we shouldn't be including communication calling. Unless we did it for a reason